recOrder/plugin/workers/acquisition_workers.py

```diff
@@ -906,6 +901,54 @@ def work(self):
         self.phase_image_emitter.emit(phase)
         self.meta_emitter.emit(meta)
 
+    def _check_exposure(self) -> None:
+        """
+        Check that all LF channels have the same exposure settings. If not, use the State0 exposure.
+
+        Parameters
+        ----------
+        settings:       (json) JSON dictionary conforming to MM SequenceSettings
+        """
+        logging.debug('Verifying exposure times...')
+        # parse exposure times
+        channel_exposures = []
+        for _, channel in enumerate(self.settings['channels']):
```

Comment:
Simplify to `for channel in self.settings['channels']:`?

Reply:
Yes that makes sense